### DISCUSSION TITLE
Comparing Input and Output texts directly

### DISCUSSION THREAD

**Discussion by vidhyasivaswamy** [truera/trulens#751] [Q&A] [answered]
Hi ,
Need a help , we have a complex prompt with preserved user history.
We want to test the input and output of the model using the feedback functions, without calling any model as such, for relevance , toxicity. Need suggestion on how to do this with this tool.

Vidhya

**Comment by joshreini1** [comment, marked as answer]
Hey @vidhyasivaswamy - what are you trying to compare?

You should be able to do this with a [custom feedback function](https://www.trulens.org/trulens_eval/custom_feedback_functions/)
